emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, user_id):
                 "metrics/daily_mean_median_speed": self.analysis_timeseries_db,
                 "inference/prediction": self.analysis_timeseries_db,
                 "inference/labels": self.analysis_timeseries_db,
+                "inference/user_label_model": self.analysis_timeseries_db,
```

Comment:
I was hoping to get this done now instead of having to deal with migrations in the future. The changes should be fairly straightforward since mongodb doesn't have any schemas. You literally need to add a new function and a new collection in `emission.core.get_database`. LMK if you don't want to tackle it and I will.